nrespipe/utils.py

```diff
@@ -536,20 +531,50 @@ def extract_signal_to_noise_from_pdf(pdf_reader: PdfFileReader, output_table: Ta
                                                               'search_text': pdf_text_to_search}})
 
 
-def get_missing_files(raw_directory, specproc_directory):
-    get_files_without_extensions_and_e00 = lambda filenames, extension: [os.path.basename(filename).replace(extension, "")[:-4]
-                                                                         for filename in filenames]
-    raw_files = get_files_without_extensions_and_e00(glob(os.path.join(raw_directory, '*e00.fits*')), '.fits.fz')
-    processed_files = get_files_without_extensions_and_e00(glob(os.path.join(specproc_directory, '*.tar.gz')), '.tar.gz')
+def query_archive_api(site, dayobs, rlevel=0, obstype='TARGET', telid='igla'):
```

Comment:
As a matter of fact, you _could_ add retries to all of these external service calls.